clicommand/pipeline_upload.go

```diff
@@ -389,7 +390,7 @@ func searchForSecrets(
 	return nil
 }
 
-func (cfg *PipelineUploadConfig) parseAndInterpolate(src string, input io.Reader, environ *env.Environment) (*pipeline.Pipeline, error) {
+func (cfg *PipelineUploadConfig) parseAndInterpolate(src string, input io.Reader, environ *env.Environment, ctx context.Context) (*pipeline.Pipeline, error) {
```

Comment:
When I tidied up context usage a while ago, I stuck to the convention that `ctx` is the first parameter (see https://pkg.go.dev/context and https://google.github.io/styleguide/go/decisions#contexts)

```suggestion
func (cfg *PipelineUploadConfig) parseAndInterpolate(ctx context.Context, src string, input io.Reader, environ *env.Environment) (*pipeline.Pipeline, error) {
```